transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java

```diff
@@ -231,6 +250,8 @@ public void release() {
         PlatformDependent.freeMemory(timeoutMemoryAddress);
     }
 
+    // The getters below are called from JNI code
+
```

Comment:
Just as a followup we should better access the fields directly from JNI as calling methods is expensive

Reply:
@normanmaurer I think it shouldn't be big issue as the methods are called when we close the RingBuffer, however it would be cleaner to access the fields directly, I'll create followup PR today